sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -14,7 +14,7 @@ Maven dependency for the Azure Key Vault Certificate client library. Add it to y
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-certificates</artifactId>
-    <version>4.1.4</version>
+    <version>4.2.0-beta.3</version>
```

Comment:
best to let READMEs point at stable release.

Reply:
below too

Reply:
Should we do that even when releasing beta?